src/main/java/io/leonis/algieba/spatial/AggregatedPotentialField.java

```diff
@@ -27,8 +27,9 @@
   @Override
   public double getPotential(final INDArray positionVector) {
     return this.potentialFields.stream()
-        .map(potentialField -> potentialField.getPotential(positionVector))
-        .reduce(0d, (total, potential) -> total + potential);
+        .mapToDouble(potentialField -> potentialField.getPotential(positionVector))
+        .reduce(Double::sum)
```

Comment:
[as suggested earlier](https://github.com/delta-leonis/algieba/pull/12#discussion_r157175512); why not use [`DoubleStream#sum`](https://docs.oracle.com/javase/8/docs/api/java/util/stream/DoubleStream.html#sum--) ?